--- 2026-10-09_14-11:/opt/sand/sand-host/officekit-ts/share/officekit/schemas/vendor/extracted/microsoft/pptx/pml-2018-4-main.xsd
+++ 2026-10-09_14-24:/opt/sand/sand-host/officekit-ts/share/officekit/schemas/vendor/extracted/microsoft/pptx/pml-2018-4-main.xsd
@@ -0,0 +1,14 @@
+<xsd:schema elementFormDefault="qualified" targetNamespace="http://schemas.microsoft.com/office/powerpoint/2018/4/main" xmlns="http://schemas.microsoft.com/office/powerpoint/2018/4/main" xmlns:p184="http://schemas.microsoft.com/office/powerpoint/2018/4/main" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <xsd:simpleType name="ST_ClassificationOutcomeType">
+    <xsd:restriction base="xsd:token">
+      <xsd:enumeration value="none"/>
+      <xsd:enumeration value="hdr"/>
+      <xsd:enumeration value="ftr"/>
+      <xsd:enumeration value="watermark"/>
+    </xsd:restriction>
+  </xsd:simpleType>
+  <xsd:complexType name="CT_ClassificationOutcome">
+    <xsd:attribute name="val" type="ST_ClassificationOutcomeType"/>
+  </xsd:complexType>
+  <xsd:element name="classification" type="CT_ClassificationOutcome"/>
+</xsd:schema>
