--- 2026-10-09_14-11:/opt/sand/sand-host/officekit-ts/share/officekit/schemas/vendor/extracted/microsoft/docx/wml-cei-2026.xsd
+++ 2026-10-09_14-24:/opt/sand/sand-host/officekit-ts/share/officekit/schemas/vendor/extracted/microsoft/docx/wml-cei-2026.xsd
@@ -0,0 +1,10 @@
+<xsd:schema xmlns:w16="http://schemas.microsoft.com/office/word/2018/wordml" xmlns:xsd="http://www.w3.org/2001/XMLSchema" blockDefault="#all" xmlns="http://schemas.microsoft.com/office/word/2026/wordml/cei" targetNamespace="http://schemas.microsoft.com/office/word/2026/wordml/cei" elementFormDefault="qualified" attributeFormDefault="qualified">
+   <xsd:import id="w16" namespace="http://schemas.microsoft.com/office/word/2018/wordml" schemaLocation="word16.xsd"/>
+   <xsd:complexType name="CT_CommentEntityInfo">
+     <xsd:sequence>
+       <xsd:element name="extLst" type="w16:CT_ExtensionList" minOccurs="0" maxOccurs="1"/>
+     </xsd:sequence>
+     <xsd:attribute name="entityType" type="xsd:unsignedInt" use="optional" default="0"/>
+   </xsd:complexType>
+   <xsd:element name="commentEntityInfo" type="CT_CommentEntityInfo"/>
+ </xsd:schema>
